Sources_v3/Workers/ChannelUpdater.swift

```diff
@@ -12,10 +12,14 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
     ///   - channelQuery: The channel query used in the request
     ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
     ///
-    func update(channelQuery: ChannelQuery<ExtraData>, completion: ((Error?) -> Void)? = nil) {
+    func update(channelQuery: ChannelQuery<ExtraData>,
+                channelCreatedCallback: ((ChannelId) -> Void)? = nil,
```

Comment:
Please update the documentation comment, too. It's very important here since this is a very specific thing we're doing here.